### PR TITLE
feature/make separator configurable

### PR DESCRIPTION
**Pull request by 89iuv**
rework for: https://github.com/catppuccin/tmux/pull/30

### CODE REVIEW

File: catppuccin.tmux [hunk 1]
```diff
@@ -61,25 +61,33 @@ main() {
   # --------=== Statusline
 
   # NOTE: Checking for the value of @catppuccin_window_tabs_enabled
-  local wt_enabled
+  local wt_enabled
```
Comment (rogeruiz) [inline catppuccin.tmux:64]:
```suggestion
  local wt_enabled
```

**Review by rogeruiz — APPROVED** [review]
lgtm 🌈 

Thanks for this @89iuv. I appreciate the effort on the documentation as well.

**Comment by nekowinston** [comment]
Hey @rogeruiz, @89iuv has reached out to be a maintainer for this repo.
Would you be happy to co-maintain it with them? They're a contributor over at catppuccin/nvim too.

**Comment by rogeruiz** [comment]
Yeah! The more the merrier.

**Pull request merged**